web/src/app/WebApp/Installers/WordPress/WordPressSetup.php

```diff
@@ -135,7 +135,7 @@ public function install(array $options = null) {
 						"'," .
 						$padding .
 						"'" .
-						addcslashes("localhost", "\\'") .
+						addcslashes($options["database_host"], "\\'") .
```

Comment:
this whole thing should have been written as
```php
$result->raw[$line_num] = 'define(' . var_export($constant,true) . ',' . var_export($options["database_host"] . ');
```
but that is not your fault, it can be fixed another time if you don't want to fix it here